iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ReactiveCapabilityCurveImpl.java

```diff
@@ -30,6 +29,9 @@ static class PointImpl implements Point {
         private double maxQ;
 
         PointImpl(double p, double minQ, double maxQ) {
+            if (minQ > maxQ) {
```

Comment:
The validation should be in ReactiveCapabilityCurveAdderImpl.addPoint
Strangely, the code is commented.

Maybe not a good idea to add this here and create *new* runtime exceptions with possible regressions.